web/src/providers/LocaleProvider.tsx

```diff
@@ -0,0 +1,102 @@
+import { Context, createContext, useContext, useEffect, useState } from 'react';
+import { useNuiEvent } from '../hooks/useNuiEvent';
+import { debugData } from '../utils/debugData';
+import { fetchNui } from '../utils/fetchNui';
+
+debugData([
+  {
+    action: 'setLocale',
+    data: {
+      ui: {
+        confirm: 'Confirm',
+        cancel: 'Cancel',
+        select: 'Select',
+        delete: 'Delete',
+        firstname: 'First Name',
+        lastname: 'Last Name',
+        gender: 'Gender',
+        male: 'Male',
+        female: 'Female',
+        non_binary: 'Non-Binary',
+        create_char: 'Create Character',
+        create_a_char: 'Create a new Character',
+        irreversible_ation: 'This action is irreversible',
+        char_location: 'Location:',
+        last_played: 'Last Played:',
+        day: 'DD',
+        month: 'MM',
+        year: 'YYYY',
+        max_chars: 'Maximum number of slots reached'
+      },
+    },
+  },
+]);
+
+interface Locale {
+  ui: {
+    confirm: string,
+    cancel: string,
+    select: string;
+    delete: string,
+    firstname: string,
+    lastname: string,
+    gender: string,
+    male: string,
+    female: string,
+    non_binary: string,
+    create_char: string,
+    create_a_char: string,
+    irreversible_ation: string,
```

Comment:
Didn't notice this the first time I reviewed, typo in word `ation`

Reply:
Oh :D

Reply:
Should be fixed in latest commit

Reply:
Wait no :DDD

Reply:
I forgot for en.json now should be fixed...